monailabel/interfaces/app.py

```diff
@@ -337,7 +337,7 @@ def on_save_label(self, image_id, label_id):
 
     def _download_tools(self):
         target = os.path.join(self.app_dir, "bin")
-        dcmqi_tools = ["segimage2itkimage", "itkimage2segimage"]
+        dcmqi_tools = ["segimage2itkimage", "itkimage2segimage", "segimage2itkimage.exe", "itkimage2segimage.exe"]
```

Comment:
I'm not sure I understand. We're using a command line tool for forward and backward conversion for DICOMSEG?